### PR TITLE
release: Bumped to version 5.0.0

### PR DESCRIPTION
**Pull request by parkhomenko**
Bumped to 5.0.0

### CODE REVIEW

**Review by faunaee — CHANGES_REQUESTED** [review]
The README.md file does not describe what the driver exception hierarchy is, or what it means for developers.

File: README.rst [hunk 1]
```diff
@@ -54,6 +54,44 @@ Basic Usage
 
     print(indexes)
 
+Exceptions handling
+---------------
+
+Since 5.0.0 release, we introduced children classes for base HttpError class,
+for example:
```
Comment (faunaee) [inline README.rst:61]:
```suggestion
With the 5.0.0 release, child classes of the base HttpError class now exist.
For example:

```

File: README.rst [hunk 2]
```diff
@@ -54,6 +54,44 @@ Basic Usage
...
+```
+class PayloadTooLarge(HttpError):
+    pass
```
Comment (faunaee) [inline README.rst:64]: Including `pass` here (and in subsequent examples) seems inappropriate. Perhaps just list the exception types and a brief description for each?

File: README.rst [hunk 3]
```diff
@@ -54,6 +54,44 @@ Basic Usage
...
+
+
+class InvalidUrlParameterError(HttpError):
+    pass
+
+
+class SchemaNotFoundError(HttpError):
+    pass
+```
+where each class corresponds to the error code from fauna database server,
```
Comment (faunaee) [inline README.rst:82]:
```suggestion
Each class corresponds to an error code from Fauna.
```

File: README.rst [hunk 4]
```diff
@@ -54,6 +54,44 @@ Basic Usage
...
+you can inspect "errors.py" file for more information on how it is implemented.
```
Comment (faunaee) [inline README.rst:83]:
```suggestion
Inspect the [`errors.py`](https://github.com/fauna/faunadb-python/blob/v5/faunadb/errors.py)
file for more information on how it is implemented.
```

File: README.rst [hunk 5]
```diff
@@ -54,6 +54,44 @@ Basic Usage
...
+
+Below you can see an example of how you can take advantage of a new exception hierarchy:
```
Comment (faunaee) [inline README.rst:85]:
```suggestion
The following example demonstrates the methods that you can access from the exception object:
```

**Review by faunaee — CHANGES_REQUESTED** [review]
A few suggestions to improve the language in the exceptions coverage.

**Comment by parkhomenko** [comment]
Opened a new one:
https://github.com/fauna/faunadb-python/pull/213

**Pull request closed without merging**